packages/ckeditor5-paste-from-office/tests/pastefromoffice.js

```diff
@@ -62,7 +62,12 @@ describe( 'PasteFromOffice', () => {
 
 				clipboard.fire( 'inputTransformation', data );
 
-				expect( data.isTransformedWithPasteFromOffice ).to.be.true;
+				expect( data._isTransformedWithPasteFromOffice ).to.be.true;
+				expect( data._parsedData ).to.have.property( 'body' );
+				expect( data._parsedData ).to.have.property( 'bodyString' );
+				expect( data._parsedData ).to.have.property( 'styles' );
+				expect( data._parsedData ).to.have.property( 'stylesString' );
```

Comment:
I'd replace it with a check that `data._parsedData` is an instance of view document fragment.

Reply:
But `data._parsedData` is not an instance of the document fragment. It is just an object with 4 properties.

I've added an additional check for `data._parsedData.documentFragment` to assert that it is an instance of view document fragment.